src/test/java/ch/uzh/ifi/hase/soprafs21/controller/UsersApiControllerTest.java

```diff
@@ -130,4 +131,76 @@ void logoutNullObject () throws Exception{
     }
 
 
+
+    @Test
+    void getUserSuccess() throws Exception {
+        UserDto mockedUserDto = getMockedUserDto();
+        given(userService.getUserDetails(Mockito.any())).willReturn(mockedUserDto);
```

Comment:
Mockito.any() does not test if the userId from the rest call is propagated to Service

Reply:
updated